test/e2e/native_histograms_test.go

```diff
@@ -336,45 +348,60 @@ func writeHistograms(ctx context.Context, now time.Time, histograms []*histogram
 	})
 }
 
-func expectedHistogramModelVector(histogram *histogram.Histogram, externalLabels map[string]string) model.Vector {
+func expectedHistogramModelVector(name string, histogram *histogram.Histogram, floatHistogram *histogram.FloatHistogram, externalLabels map[string]string) model.Vector {
```

Comment:
Nit: can we rename `name` to `metricName`?

Reply:
Yes. Done.